tests/unit/models/test_theme_mapping.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from consultation_analyser.consultations import models
+from consultation_analyser.factories import (
+    AnswerFactory,
+    ExecutionRunFactory,
+    FrameworkFactory,
+    QuestionPartFactory,
+    ThemeFactory,
+    ThemeMappingFactory,
+)
+
+
+@pytest.mark.django_db
+def test_get_latest_theme_mappings_for_question_part_returns_latest_mappings():
+    question_part = QuestionPartFactory()
+    answer = AnswerFactory(question_part=question_part)
+
+    # First theme mapping execution run
+    first_framework = FrameworkFactory(question_part=question_part)
+    theme1 = ThemeFactory(framework=first_framework)
+    first_execution_run = ExecutionRunFactory(type=models.ExecutionRun.TaskType.THEME_MAPPING)
+    ThemeMappingFactory(answer=answer, theme=theme1, execution_run=first_execution_run)
+
+    # Second theme mapping execution run
```

Comment:
We're almost writing app code here! It would be nice to be able to hit a "map_themes" method or something and have it actually run the pipeline (a fake pipeline in tests, of course) and write the appropriate records?

Reply:
I decided not to do this in the interests of getting this PR over soon... (but happy to be overruled). Thought it best to do this when we add the pipeline anyway (or if it turns out to be useful in other tests).